content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +156,47 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
+
+To control which sysctls can be set in pods, specify either the
+`forbiddenSysctls` and `allowedUnsafeSysctls` fields in the PodSecurityPolicy.
+
+By default, all safe sysctls in the whitelist are allowed.
+
+Both `forbiddenSysctls` and `allowedUnsafeSysctls` are lists of plain sysctl names
+or sysctl patterns (which end with `*`). The string `*` matches all sysctls.
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The `forbiddenSysctls` field excludes specific sysctls, and can include a
+combination of safe and unsafe ones. To forbid setting any sysctls, use `*` on
+its own.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+If you specify any unsafe sysctl in the `allowedUnsafeSysctls` field and it is
+not present in the `forbiddenSysctls` field, that sysctl can be set in the
+PodSecurityPolicy. To allow all unsafe sysctls to be set (except for those explicitly forbidden by `forbiddenSysctls`), use `*` on its own.
 
-Here is an example, it authorizes binding user creating pod with corresponding sysctls.
+Be careful not to configure these two fields such that there is overlap.
+
+{{< warning >}}
+**Warning**: If you use the `--allowed-unsafe-sysctls` flag when starting the
+pod and the value of the flag conflicts with the contents of the
+`allowedUnsafeSysctls` field in the PodSecurityPolicy, the pod will fail to
```

Comment:
what does "conflicts with the contents of the `allowedUnsafeSysctls` field" mean? The is no directly relation between the two. This would be correct: If you whitelist unsafe sysctls via the `allowedUnsafeSysctls` field in a PSP, any pod using such a sysctl will fail to start if the sysctl is not whitelisted via the `--allowed-unsafe-sysctls` kubelet flag as well on that node.